People/DanScime/Print/HTML-Print.hs

```diff
@@ -0,0 +1,491 @@
+module Language.Drasil.HTML.Print(genHTML) where
```

Comment:
Do you really want this file in this PR?

Reply:
Don't need anymore. Will remove.